loadtest/store/memstore/store.go

```diff
@@ -52,6 +52,7 @@ type MemStore struct {
 	featureFlags        map[string]bool
 	report              *model.PerformanceReport
 	channelBookmarks    map[string]*model.ChannelBookmarkWithFileInfo
+	scheduledPosts      map[string]map[string]*model.ScheduledPost
```

Comment:
Also, please update the `(s *MemStore) Clear()` method to clear this field as well.

Reply:
Done.